spring-cql/src/test/java/org/springframework/cassandra/config/CassandraCqlSessionFactoryBeanUnitTests.java

```diff
@@ -0,0 +1,349 @@
+/*
+ *  Copyright 2013-2016 the original author or authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cassandra.config;
+
+import static org.hamcrest.Matchers.equalTo;
```

Comment:
Not a big deal, but let's stick to the [Spring Data Formatter settings](https://github.com/spring-projects/spring-data-cassandra/blob/master/etc/eclipse-formatting.xml). Static imports are imported with using wildcards then.

![bildschirmfoto 2016-05-11 um 09 17 28](https://cloud.githubusercontent.com/assets/1035015/15172928/3cbb986c-1759-11e6-859f-bfb1252f5f0f.png)